setup.py

```diff
@@ -86,7 +86,7 @@ def _get_dbt_core_version():
     },
     install_requires=[
         "dbt-core~={}".format(dbt_version),
-        "trino==0.315.0",
+        "trino==0.317.0",
```

Comment:
Are all these changes related to supporting grants or could they be extracted to smaller unrelated PRs ?

Reply:
We need 0.317.0 as in that version we introduced roles support.

We even need the not yet released 0.318.0 as we need another fix as prepared statements are not yet fully working in 0.317.0 (see the CI failures and https://github.com/trinodb/trino-python-client/pull/249)

Reply:
Sorry for the confusion. The PR contains 6 commits - which seemed to me (correct me if I'm wrong) are not all related to one another. Could you extract the commits which are unrelated to the grants to one or more preparatory PRs ?

Reply:
I'll make a PR with only Trino, Starburst and trino-python-client version upgrade but I'm waiting for the 0.318.0 version to be released.